getting-started/codecs.md

```diff
@@ -1,39 +1,38 @@
 ---
 title: Codecs
 description: Understanding data. De- and encode data from wire formats.
-hide_table_of_contents: true
+hide_table_of_contents: false
 ---
 
 ### Concept
 
 Tremor connects to the external systems using connectors.
 
-Connectors that integrate tremor with upstream systems from where tremor is typically
-ingesting data are called `Onramps`.
+Connectors that integrate Tremor with upstream systems from where Tremor is typically ingesting data are called `Onramps`.
 
-Connectors that integrate tremor with downstream systems where tremor is typically
-publishing or contributing data to are called `Offramps`.
+Connectors that integrate Tremor with downstream systems where Tremor is typically publishing or contributing data to are called `Offramps`.
 
-`Onramps` and `Offramps` use `codecs` to transform the external wire form of connected
-system participants into a structured internal value type tremor understands semantically.
+`Onramps` and `Offramps` use `codecs` to transform the external wire form of connected system participants into a structured internal value type Tremor understands semantically.
 
 Tremor's internal type system is JSON-like.
 
-`Onramps` and `Offramps` support `preprocessors` and `postprocessors`. External data ingested into tremor
-via `Onramps` can be pre-processed through multiple transfomers before a code is applied to convert the
-data into tremor internal form. Preprocessors are configured as a chain of transformations. Postprocessors
-are applied to values leaving tremor after a codec transform them from tremor internal form to wire
-form. Postprocessors are configured as a chain of transformations.
+`Onramps` and `Offramps` support `preprocessors` and `postprocessors`. External data ingested into Tremor via `Onramps` can be pre-processed through multiple transfomers before a code is applied to convert the data into Tremor-internal form. Preprocessors are configured as a chain of transformations. Postprocessors
+are applied to values leaving Tremor after a codec transforms them from Tremor internal form to wire form. Postprocessors are configured as a chain of transformations.
 
+<<<<<<< HEAD
```

Comment:
there is a merge artifact in here, I think we want to remove that :) I think we want the relative path